sorting/strand_sort.cpp

```diff
@@ -0,0 +1,44 @@
+#include <iostream>
```

Comment:
See #962 as a reference on how to document the first comment block.
```suggestion
/**
 * @file
 * @brief description of algorithm
 *
 * @details
 * <full-description here>
 *
 * @author [Mertcan Davulcu](https://github.com/mertcandav)
 */
#include <iostream>
```